### DISCUSSION TITLE
matplotlib__python的繪圖好幫手

### DISCUSSION THREAD

**Discussion by hapiwangy** [1101-GS4719-C/Python-Community-GS4719-C#407] [General]
相信有許多的同學在python微積分當中有接觸到這類作圖相關的語法。這篇文章除了帶我們一步一步地畫出特定資料的圖片之外，也教會了我如何對已經做好的圖片增加顏色、改變線條樣式、設定透明度、標題、刻度顏色的更改等等......。這讓我在將來報告統計圖表的時候除了excel、word之外又有了更多的選擇。
[https://medium.com/%E6%95%B8%E6%93%9A%E4%B8%8D%E6%AD%A2-not-only-data/%E6%B7%B1%E5%85%A5%E6%B7%BA%E5%87%BA-python-%E8%A6%96%E8%A6%BA%E5%8C%96%E5%A5%97%E4%BB%B6-matplotlib-seaborn-%E6%A0%B8%E5%BF%83%E6%8C%87%E5%8D%97%E8%88%87%E7%B9%AA%E8%A3%BD-44a47458912](https://medium.com/%E6%95%B8%E6%93%9A%E4%B8%8D%E6%AD%A2-not-only-data/%E6%B7%B1%E5%85%A5%E6%B7%BA%E5%87%BA-python-%E8%A6%96%E8%A6%BA%E5%8C%96%E5%A5%97%E4%BB%B6-matplotlib-seaborn-%E6%A0%B8%E5%BF%83%E6%8C%87%E5%8D%97%E8%88%87%E7%B9%AA%E8%A3%BD-44a47458912)

**Comment by ThomasMeng0515** [comment]
matpotlib 有時候在做機器學習的應用時，常會搭配seaborn模組將資料視覺化，所以再了解matplotlib之後可以再學習seaborn相關知識，讓資料能更生動地呈現出來。

**Comment by pekoranoinu** [comment]
微積分的python就是照著範例解，但我對python還是不太會用，感謝你的分享。

**Comment by bloodtrail0511** [comment]
微積分教的python只是照著範例打一樣的句子，然後把函數改掉而已。這篇文章讓我學到matlab的各種設定，以及微積分的範例程式為甚麼要打那些指令了，感謝分享

**Comment by jason79461385** [comment]
之前幫朋友畫很多函式圖的時候，就有用過呢，特別去找怎麼分顏色等

**Comment by treeann** [comment]
之前製作表格常常使用 謝謝樓主

**Comment by ghost** [comment]
這東西ˋ真的常用ww

**Comment by sys511613** [comment]
之前的作業也有用過matplotlib做資料視覺化，可以說是方便又不會太難

**Comment by alexthebigpp** [comment]
matplotlib真的是繪圖必備的好工具。

**Comment by yenwei12138** [comment]
微積分的python有點過於簡單，用這個解感覺是殺雞用牛刀，但還是感謝分享~

**Comment by 110409538** [comment]
微積分的程式作業很像也有用到

**Comment by Lynn12378** [comment]
微積分作業有做到，但感覺不是很懂怎麼運作的。感謝樓主分享

**Comment by Hchejxnc** [comment]
matplotlib真的是繪圖必備的好工具。

**Comment by Zoe06060606** [comment]
之前上過機器學習的課程的時候也很常用到這個，甚至圖像辨識的課也在用，真的要好好熟悉matplotlib這塊的語法

**Comment by s96015welson** [comment]
在上機器學習時有用過這個函示庫，我認為比excel好用很多，搭配python上其他資料處理的函式庫一起使用，在整理大量數據時可以事半功倍。雖然在學習使用上稍微有點門檻，不過通常是想好要如何做時發現不會用，但上網查一下有熱心的網友分享教學了。另外若是有用過matlab的人使用這個很快就會上手了，有許多類似的地方。

**Comment by Kotori0501** [comment]
matplotlib這模組是真的好用，現在大氣領域中，可以利用局部放大圖來看海洋溫度與氣壓間的剖面圖，並將表面海洋放大來看，又可以更加的知道表面海洋的資訊。

**Comment by ru0372572** [comment]
系上老師也常常用matplotlib來畫圖，mpl_toolkits.mplot3d是個可以畫3D圖的好用工具！Python各種模組真的是超級好用，而且短短幾行就可以做低階語言要打超多行的事，對於學習能有簡略的基本認識

**Comment by AnAnistired** [comment]
微積分的python作業真的滿沒意義的，很多人都是第一次接觸python，而且特別開設的python竟然要分批報名參加，輪到我上的時候已經是第三份作業了，前面要不是有同學的可以抄，根本就寫不出來🥲

**Comment by RogelioKG** [comment]
微積分的 Python 作業確實是蠻過分的 (雖然我是寫的蠻開心的？
我是一年前接觸 matplotlib 而且做了大量筆記，
才有辦法行雲流水的畫各種酷酷的圖。
但身邊的室友，包含大部分小大一都沒有接觸過程式語言。
還是希望學校要明白，如果要出一份作業，除了自編教材，最好也要先上課程才行。
